Library/Homebrew/test/test_dependency_collector.rb

```diff
@@ -39,7 +39,8 @@ def test_dependency_tags
 
   def test_requirement_creation
     @d.add :x11
-    assert_instance_of X11Requirement, find_requirement(X11Requirement)
+    req = OS.mac? ? X11Requirement : XorgRequirement
+    assert_instance_of req, find_requirement(req)
```

Comment:
Should these all be either `find_requirement(:x11)` or `find_requirement("x11")`? (I haven't checked.)